.claude/commands/orchconverge.md

```
@@ -0,0 +1,490 @@
+```bash
+/orchconverge <goal>                           # Start autonomous convergence with default limits
+/orchconverge <goal> --max-attempts 15        # Custom attempt limit (default: 10)
+/orchconverge <goal> --max-hours 6            # Custom time limit (default: 3 hours)
+/orchconverge <goal> --interval 5             # Convergence attempt interval in minutes (default: 5)
+```
+
+## Command Architecture - Integration with Existing /orch System
+
+**`/orchconverge` = Existing Orchestration System + Convergence Agent + Final Review Workflow**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)